macros/GroupData.json

```diff
@@ -1969,6 +1969,7 @@
                             "VRPose",
                             "VREyeParameters",
                             "VRFieldOfView",
+                            "VRLayer"
```

Comment:
In the newest spec, it is not defined as a dictionary, and even has its own constructor: https://w3c.github.io/webvr/#interface-vrlayer